safe-global/safe-wallet-web
# [EPIC] SEP #5 - Update header/widget

## What it solves

Resolves missing `type`/staging URL for SEP #5 claiming data

## How this PR fixes it

The forthcoming claiming data for SEP #5 will be tagged under `user_v2`. This adds the new `type`, as well as the staging URL.

## How to test it

Open the PR deployment on a Safe that is part of the [test allocation](https://docs.google.com/spreadsheets/d/1_T-PfjItlejyYfWlUfvQGXCtF_wCPb_508xi1NC7edQ/edit#gid=216932284) and observe the increased allocation in the header/governance app widget.

## Checklist
